api_checkout.md

```diff
@@ -59,8 +72,23 @@ Example:
 ```
 
 ### Checkout Info
+
 A signed document with mandatory price calculation and available payment methods.
 
+#### Line Item
+
+A Line Item
+
+| Parameter           | Type         | Default | Description                                                          |
+|---------------------|--------------|---------|----------------------------------------------------------------------|
+| sku                 | string       |         | SKU of the product                                                   |
+| amount              | int          |         | Number of products                                                   |
```

Comment:
Vielleicht erwähnt man hier noch, dass es sich um `number of products / packages` handelt, so dass deutlich wird, dass ein Bundle aus 6 Produkten trotzdem amount = 1 ist.